test/integration/patients/worklist/worklist.js

```diff
@@ -1823,32 +1816,34 @@ context('worklist page', function() {
       role: getRelationship(roleEmployee),
     };
 
+    const testCurrentClinician = getCurrentClinician({ relationships });
+
     cy
       .routeWorkspaceClinicians(fx => {
         fx.data = [
-          getCurrentClinician({ relationships }),
+          testCurrentClinician,
           getClinician({
             id: 'test-clinician',
             attributes: { name: 'Test Clinician' },
             relationships,
           }),
           getClinician({
-            id: '1',
+            id: uuid(),
```

Comment:
not sure, but it seems like part of a refactor will be just making `getClinician` auto use `uuid`..  or perhaps it's `getNewClinician` or something..  I understand that with clinicians because they're bootstrapped, many places will need to `getClinician` and pull from the list of ids in the workspace.  We should be able to accommodate that, and also not require passing a `uuid` to every helper when we're setting the bootstrapped clinicians.